src/System.Text.Encoding/tests/Encoding/EncodingVirtualTests.cs

```diff
@@ -0,0 +1,44 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Thanks for some these. But you need to add this file to the .csproj, otherwise these won't be included/run.

Reply:
Thanks for reminding. Why aren't there "default includes" in these projects?